### PR TITLE
Retail

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated retail page

## QA

- `./run`
- Go to /programmes/retail

## Screenshots

screenshot

### CODE REVIEW

File: templates/programmes/retail.html [hunk 1]
```diff
@@ -7,79 +7,79 @@
...
     </div>
-</div>
-<div class="row no-border no-padding-top">
-    <div class="twelve-col">
-        <img src="{{ STATIC_URL }}img/map-retail.svg" width="100%" alt="World map" class="priority-0" />
+  </div>
+</section>
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <img src="{{ STATIC_URL }}img/map-retail.svg" width="100%" alt="World map" class="priority-0" />
```
Comment (anthonydillon) [inline templates/programmes/retail.html]: `s/priority-0/u-hide--small`

File: templates/programmes/retail.html [hunk 2]
```diff
@@ -7,79 +7,79 @@
...
-    <div class="eight-col append-four">
-        <p>We&rsquo;ve been working with partners like Dell, HP  and Lenovo for years to ensure that Ubuntu devices reach market as effectively as possible. With the help of these partners, Ubuntu machines are distributed around the world and can be found in both their exclusive and partner stores.</p>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="u-align--center">
+        <img src="https://assets.ubuntu.com/v1/68713fd5-logo-lenovo.svg" alt="Lenovo logo" width="200" />
+      </div>
+      <p>Canonical has been shipping Ubuntu on Lenovo&rsquo;s popular Think Center and Idea Center machines for over 12 months to Brazil and China through their in-country websites.</p>
+      <p><a href="https://www3.lenovo.com/gb/en/?LandingPage=/All/US/Portals/Consumer/country-selector" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Retail Page Link', 'eventAction' : 'Lenovo', 'eventLabel' : 'Shop for Ubuntu with Lenovo link', 'eventValue' : undefined });">Shop for Ubuntu with&nbsp;Lenovo&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/programmes/retail.html]: External link

File: templates/programmes/retail.html [hunk 3]
```diff
@@ -7,79 +7,79 @@
...
-                <p><a href="https://www.amazon.com/s/ref=nb_sb_noss_1?url=search-alias%3Dcomputers&field-keywords=ubuntu" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Retail Page Link', 'eventAction' : 'Amazon', 'eventLabel' : 'Shop for Ubuntu with Amazon link', 'eventValue' : undefined });">Shop for Ubuntu with&nbsp;Amazon&nbsp;&rsaquo;</a></p>
-            </div>
-        </div>
+    <div class="col-6 p-card">
+      <div class="u-align--center">
+        <img src="{{ STATIC_URL }}img/logos/logo-amazon.jpg" alt="Amazon logo" width="200" />
+      </div>
+      <p>Amazon are a keen partner in selling Ubuntu preloaded machines on both their French website Amazon.fr and in the US on Amazon.com.</p>
+      <p><a href="https://www.amazon.com/s/ref=nb_sb_noss_1?url=search-alias%3Dcomputers&field-keywords=ubuntu" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Retail Page Link', 'eventAction' : 'Amazon', 'eventLabel' : 'Shop for Ubuntu with Amazon link', 'eventValue' : undefined });">Shop for Ubuntu with&nbsp;Amazon&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/programmes/retail.html]: External

**Comment by kwm14** [comment]
Nice work @deadlight 👍

**Comment by anthonydillon** [comment]
@deadlight still missing the external link classes

**Pull request merged**